addons/pos_details/static/src/xml/details.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="SaleDetailsButton">
+        <div class="oe_status">
+            <div class="js_connected oe_icon">
+                <i class='fa fa-fw fa-print'></i>
+            </div>
+        </div>
+    </t>
+
+    <t t-name="SaleDetailsReport">
+        <receipt align='center' width='40' value-thousands-separator='' >
+            <t t-if='company.logo'>
+                <img t-att-src='company.logo' />
```

Comment:
I don't think the logo is loaded on res.company records. The base64 encoded logo (which you need here) is in `posmodel.company_logo_base64`.